input/text/src/commonMain/kotlin/majestic/TextField.kt

```diff
@@ -163,7 +166,7 @@ fun TextField(
     Column(modifier = modifier) {
         label()
         OutlinedTextField(
-            modifier = Modifier.fillMaxWidth(),
+            modifier = Modifier.fillMaxWidth().then(if (maxLines > 56) Modifier.height(maxLines.dp) else Modifier.wrapContentHeight()),
```

Comment:
I don't think this is a good idea. Because this modifier here is going to set the height of the field to 60dp for a 60dp maxLines text field. However, that might not be the behavoir someone needs.

this is very specific to a use case, that i think it should be configured on the consumer side, or we should find another alternative to your problem